build/linux/Makefile

```diff
@@ -1,5 +1,5 @@
 TOP:=../..
-CXX?=clang++-3.9
+CXX=clang++-3.8
```

Comment:
Ah, so that's what the ? means...I just found that the build didn't work if I left the ? there. If it will work with me just manually passing a flag instead of changing this for everyone, that seems like the clearly better way to go.